crates/holochain_types/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+# [Unreleased]
+
+## Changed
+- BREAKING: All references to `"uuid"` in the context of DNA has been renamed to `"uid"` to reflect that these IDs are not universally unique, but merely unique with regards to the zome code (the genotype) [#727](https://github.com/holochain/holochain/pull/727)
```

Comment:
It's a bit tricky because some of these affect multiple crates. Should we be duplicating changelog items which affect multiple?

Reply:
I missed the fact that the changelog gets aggregated. So this looks fine as it is.